diff --git a/CMakeLists.txt b/CMakeLists.txt
index f1ab623..55d3ac2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -72,7 +72,7 @@ IF (${VERSION}.x STREQUAL ".x")
    MESSAGE("Extracting git software version")
    execute_process(COMMAND bash "-c" "git rev-list --branches HEAD | wc -l | tr -d ' ' | tr -d '\n'" OUTPUT_VARIABLE GIT_VERSION)
    SET(MINOR_VERSION 3)
-   SET(BUILD_NUMBER ${GIT_VERSION})
+   SET(BUILD_NUMBER 0 )
    SET(VERSION ${MAJOR_VERSION}.${MINOR_VERSION}.${BUILD_NUMBER}) 
 ENDIF()
 MESSAGE("Software Version: ${VERSION}")
